project/BinaryIncompatibilities.scala

```diff
@@ -20,6 +20,40 @@ object BinaryIncompatibilities {
   val CLI = Seq(
   )
 
+  val Library = Seq(
+      // private[runtime], not an issue
+      ProblemFilters.exclude[MissingMethodProblem](
+          "scala.scalajs.runtime.RuntimeLong.TWO_PWR_16_DBL"),
+      ProblemFilters.exclude[MissingMethodProblem](
+          "scala.scalajs.runtime.RuntimeLong.TWO_PWR_22_DBL"),
+      ProblemFilters.exclude[MissingMethodProblem](
+          "scala.scalajs.runtime.RuntimeLong.TWO_PWR_31_DBL"),
+      ProblemFilters.exclude[MissingMethodProblem](
+          "scala.scalajs.runtime.RuntimeLong.TWO_PWR_32_DBL"),
+      ProblemFilters.exclude[MissingMethodProblem](
+          "scala.scalajs.runtime.RuntimeLong.TWO_PWR_44_DBL"),
+      ProblemFilters.exclude[MissingMethodProblem](
+          "scala.scalajs.runtime.RuntimeLong.TWO_PWR_63_DBL"),
+      ProblemFilters.exclude[MissingMethodProblem](
+          "scala.scalajs.runtime.RuntimeLong.TenPow9"),
+      ProblemFilters.exclude[MissingMethodProblem](
+          "scala.scalajs.runtime.RuntimeLong.SIGN_BIT"),
+      ProblemFilters.exclude[MissingMethodProblem](
+          "scala.scalajs.runtime.RuntimeLong.SIGN_BIT_VALUE"),
+      ProblemFilters.exclude[MissingMethodProblem](
+          "scala.scalajs.runtime.RuntimeLong.TWO_PWR_15_DBL"),
+
+      // Synthetic methods inside RuntimeLong, not an issue
```

Comment:
Stuff. You never know what Scalac is going to emit for stuff. But they're marked as "synthetic", and that, coupled with the fact they apparently look like they come for previously private methods, generally means it's safe to ignore the changes.